File: web3rpc/rpc-specs/code-samples/python/eth/transaction/getTransactionByBlockNumberAndIndex.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0x27"
transactionIndex = "0x0"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_transaction_by_block_number_and_index(blockTag, transactionIndex)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/transaction/getTransactionByBlockNumberAndIndex.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const blockNumber = 118593751;
  const index = "0x0";
  provider.eth
    .getTransactionByBlockNumberAndIndex(
      blockNumber,
      index,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
